lib/bemxjst/index.js

```diff
@@ -8,7 +8,7 @@ var ClassBuilder = require('./class-builder').ClassBuilder;
 var utils = require('./utils');
 
 function BEMXJST(options) {
-  this.options = options || {};
+  this.options = options;
```

Comment:
Because of [`options` is always object](https://github.com/bem/bem-xjst/blob/master/lib/compiler.js#L12).